sdk/requirements.txt

```diff
@@ -4,7 +4,7 @@ fire
 google-auth>=1.11.0
 google-cloud-storage>=1.19.0
 mlflow>=1.2.0,<=1.23.0
-numpy
+numpy<1.24.0
```

Comment:
This change is needed due to a breaking change from numpy v1.24.0 that is incompatible with our pinned versions of mlflow due to the deprecation of several common numpy aliases (see the last bullet point here: https://numpy.org/doc/stable/release/1.24.0-notes.html#expired-deprecations).

Reply:
Added that in the file :)